src/main/java/hometaskStream/Book.java

```diff
@@ -0,0 +1,134 @@
+package hometaskStream;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class Book {
+    private String name;
+    private int price;
+
+    public Book() {
+        this.price = getRandomBookPrice();
+        this.name = getRandomBookName();
+    }
+
+    public Book(String name, int price) {
+        this.name = name;
+        this.price = price;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Book book = (Book) o;
+        return price == book.price && name.equals(book.name);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(name, price);
+    }
+
+    @Override
+    public String toString() {
+        return "Book name is " + this.name.toUpperCase() + " Book price is " + this.price + "$";
```

Comment:
book name should not be fully in upper case, please capitalize its name during the generation of random book name (getRandomBookName())

Reply:
thanks, done

Reply:
still not capitalized, it's all in upper case. capitalization - is when the first letter is in upper case, all other - in lower case

Reply:
` return buffer.substring(0, 1).toUpperCase() + buffer.substring(1);` was added to getRandomBookName method